content/frontend-web/css-sass/webpack-configuration-for-sass-only.md

```diff
@@ -17,7 +17,7 @@ $ yarn init -y
 
 # Add required packages to the project.
 ```sh
-yarn add --dev \
+$ yarn add --dev \
```

Comment:
@mildronize it is good one to show you are not root user but it is a little bit annoying because you can't copy and paste.
Many times, you need to remote $ from a terminal before executing a command. LoL.